docs/whitepapers/shortpaper.mdx

```diff
@@ -0,0 +1,641 @@
+        <td>Token Category
+        </td>
+        <td>3 months
+        </td>
+        <td>9 months
+        </td>
+        <td>12 months
+        </td>
+        <td>24 months
+        </td>
+    </tr>
+    <tr>
+        <td>Category 1
+        </td>
+        <td>1.00 (0% boost)
+        </td>
+        <td>1.10 (10%)
+        </td>
+        <td>1.20 (20%)
+        </td>
+        <td>1.40 (40%)
+        </td>
+    </tr>
+    <tr>
+        <td>Category 2
+        </td>
+        <td>1.00 (0% boost)
+        </td>
+        <td>1.02 (2%)
+        </td>
+        <td>1.04 (4%)
+        </td>
+        <td>1.08 (8%)
+        </td>
+    </tr>
+    <tr>
+        <td>Category 3
+        </td>
+        <td>1.00 (0% boost)
+        </td>
+        <td>1.01 (1%)
+        </td>
+        <td>1.02 (2%)
+        </td>
+        <td>1.04 (4%)
+        </td>
+    </tr>
+</table>
+
+*Table 4: Calculating ICBAM Mining Duration Bonus (MDB) using Token Category and Lock Duration*
+
+$$
+MDB = Σ(A, B)
+$$
+
+*Equation 5: Calculating the total Mining Duration Bonus (MDB)*
+
+The MDB is the sum of:
+* A: ICBAM (token mining) MDB, which is DHX Mining, with a maximum value of 0.4 (e.g. 40%, which is shown as 1.4 in Table 4), and;
+* B: Non-ICBAM MDB also with a maximum value of 0.4. This excludes hardware mining since that is covered separately by MHB.
+
+##### Mining Hardware Bonus (MHB)
+
+Different hardware (HW) devices connected with the network have a MHB specific to the category of hardware connected.
+
+If you have a MHB, then this would add a boost to your total mPower, applied to the total mPower including all other bonuses. For example, if you have a Category 1 piece of hardware, the bonus would be applied as:
+
+$$
+min(mPower, HW#  1,000,000)
+$$
+
+*Equation 6: Calculating the Mining Hardware Bonus (MHB)*
+
+Each mPower unit can be boosted only by 1 HW device, e.g. if we have 200k mPower from our Category 1 tokens and two Category 1 HW devices then the total mPower will equal 400k (since a MF of 2 is applied from Table 5), as only the 1st HW device may be boosted ***.
+
+<table>
+    <tr>
+        <td>Hardware Category
+        </td>
+        <td>Mining Factor (MF)
+        </td>
+        <td>Max amount of mPower that can apply per gateway (maxA)
+        </td>
+    </tr>
+    <tr>
+        <td>Category 1
+        </td>
+        <td>2
+        </td>
+        <td>
+            1 millionmin(dhxExchangeRate, 1)
+        </td>
+    </tr>
+    <tr>
+        <td>Category 2
+        </td>
+        <td>1.5
+        </td>
+        <td>
+            500,000min(dhxExchangeRate, 1)
+        </td>
+    </tr>
+    <tr>
+        <td>Category 3
+        </td>
+        <td>1.25
+        </td>
+        <td>
+            250,000 min(dhxExchangeRate, 1)
+        </td>
+    </tr>
+</table>
+
+*Table 5: Calculating Mining Hardware Bonus (MHB) using Hardware Category*
+
+
+:::caution
+Equation formatting is wrong
+:::
+$$
+MHB = (MB MDB )( MF * HW# - 1)		if  MHB < (maxA * HW#)
```

Comment:
what is `(MB MDB )` is that a multiplication?
should the multiplication be shown more clearly as `(MB MDB )( MF * HW# - 1)` be `(MB MDB ) * ( MF * HW# - 1)`?